connman/src/service.c

```diff
@@ -3433,6 +3433,27 @@ static void append_properties(DBusMessageIter *dict, dbus_bool_t limited,
 				GET_DEFAULT_ACCESS_ACCESS);
 }
 
+static void append_properties_updated(DBusMessageIter *dict,
+	struct connman_service *service)
```

Comment:
[Coding style M4](https://github.com/sailfishos/connman/blob/master/connman/doc/coding-style.txt#L78) instructs to intend these new lines to the right.

Reply:
Thanks a lot for the review, I have fixed that

Reply:
Similar comment here as in https://github.com/sailfishos/connman/pull/3#discussion_r688444799

Reply:
Thanks a lot. I have fixed that.

Reply:
Great!